usr/share/rear/format/USB/default/200_check_usb_layout.sh

```diff
@@ -24,6 +21,10 @@ elif [[ "$TEMP_USB_DEVICE" && -d "/sys/block/$TEMP_USB_DEVICE" ]] ; then
     RAW_USB_DEVICE="/dev/$( my_udevinfo -q name -p "$TEMP_USB_DEVICE" )"
 elif [[ -z "$TEMP_USB_DEVICE" ]] ; then
     RAW_USB_DEVICE="/dev/$( my_udevinfo -q name -n "$REAL_USB_DEVICE" )"
+elif [[ -n "$( lsblk -r -o NAME,KNAME,TYPE,PKNAME | grep "$(basename $REAL_USB_DEVICE)" | grep part )" ]]; then
+    RAW_USB_DEVICE="/dev/$( lsblk -r -o NAME,KNAME,TYPE,PKNAME | grep "$(basename $REAL_USB_DEVICE)" | grep part | awk '{print $4}' | uniq )"
```

Comment:
Could you add an explanatory comment
(preferably with a real-world example)
what those
```
... | grep ... | grep ... | awk ... | uniq
```
pipe actually does because from plain looking
at the code at least I don't understand, cf.
https://github.com/rear/rear/wiki/Coding-Style#code-must-be-easy-to-read